smoke-test/tests/cypress/example_to_datahub_rest.yml

```diff
@@ -0,0 +1,11 @@
+# see https://datahubproject.io/docs/metadata-ingestion/source_docs/file for complete documentation
+source:
+  type: "file"
+  config:
+    filename: "./data.json"
+
+# see https://datahubproject.io/docs/metadata-ingestion/sink_docs/datahub for complete documentation
+sink:
+  type: "datahub-rest"
+  config:
+    server: "http://localhost:8080"
```

Comment:
nice, thanks for adding this!

Reply:
hahaha

Reply:
this was a mistake, but should I leave it here?

Reply:
lol